secp256k1-sys/src/lib.rs

```diff
@@ -629,7 +630,8 @@ pub unsafe extern "C" fn rustsecp256k1_v0_3_1_default_error_callback_fn(message:
     panic!("[libsecp256k1] internal consistency check failed {}", msg);
 }
 
-
+#[no_mangle]
+#[cfg(not(feature = "external-symbols"))]
```

Comment:
Why do we need/want this?

Reply:
ok, I get that the `#[cfg(not(feature = "external-symbols"))]` is to prevent it from complaining on unused function
but why the no_mangle? I don't think that's correct